### ISSUE TITLE
[FEATURE-REQUEST]  is it still supported 

### ISSUE THREAD

**Issue by Sandy4321** [vaexio/vaex#2413]
**Description**
 is it still supported 
I do not see changes for last 6 months 
by the way can it be used on Windows oS?

**Comment by arunpersaud** [comment]
yes, would be nice to get a quick update. I have seen some comments on some issues, but no commit on the main branch?!
